GitUI/BitmapExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿#nullable enable
+
+using System;
+using System.Drawing;
+
+namespace GitUI
+{
+    public static class BitmapExtensions
+    {
+        public static Icon BitmapToIcon(this Bitmap bitmap)
```

Comment:
💡 As an extension method, `Bitmap` seems redundant.

```suggestion
        public static Icon ToIcon(this Bitmap bitmap)
```

Reply:
I forgot to push...